packages/core/src/components/Pagination.story.tsx

```diff
@@ -25,6 +26,26 @@ storiesOf('Core/Pagination', module)
       onPrevious={action('onPrevious')}
     />
   ))
+  .add('With label.', () => (
+    <Pagination
+      hasPrev
+      hasNext
+      pageLabel={T.phrase('Photo', {}, 'Label for photo pagination')}
+      page={2}
+      onNext={action('onNext')}
+      onPrevious={action('onPrevious')}
+    />
+  ))
+  .add('With no label.', () => (
+    <Pagination
+      hasPrev
+      hasNext
+      pageLabel=""
```

Comment:
since the type is `string | undefined` (optional), let's removed `pageLabel=""` from the example.

Reply:
So I ran into some trouble here while trying to avoid a breaking change. 😬 If we want no label, but the default is "Page", is there another way to have no label? Maybe a `noLabel` prop, or is that redundant?

Reply:
aaah i see! so `undefined` can get the defaultProp `Page` and when `pageLabel=""`, do the translation phrases without it as a variable